# Automatically generated by Pynguin.
import thefuck.logs as module_0

def test_case_0():
    try:
        int_0 = 2559
        var_0 = module_0.exception(int_0, int_0)
    except BaseException:
        pass

def test_case_1():
    try:
        float_0 = 3600.64
        list_0 = [float_0, float_0, float_0]
        tuple_0 = None
        var_0 = module_0.rule_failed(list_0, tuple_0)
    except BaseException:
        pass

def test_case_2():
    try:
        int_0 = -2229
        var_0 = module_0.show_corrected_command(int_0)
    except BaseException:
        pass

def test_case_3():
    try:
        bytes_0 = b'\xac\x83\xd7\xc3o\xa4\x0eA?$\xf6\xa3'
        var_0 = module_0.confirm_text(bytes_0)
    except BaseException:
        pass

def test_case_4():
    try:
        str_0 = 'B*4tn%"4'
        var_0 = module_0.how_to_configure_alias(str_0)
    except BaseException:
        pass

def test_case_5():
    try:
        str_0 = '\r\r'
        var_0 = module_0.already_configured(str_0)
    except BaseException:
        pass

def test_case_6():
    try:
        bytes_0 = b'\x9b\xec\xe8.YnI\x07'
        var_0 = module_0.configured_successfully(bytes_0)
    except BaseException:
        pass